VizNetworkApp/Transport/NetworkTransporter.swift

```diff
@@ -35,10 +35,10 @@ class NetworkManager {
             group.enter()
             
             let dataTask = URLSession.shared.dataTask(with: request) { data, response, error in
```

Comment:
request needs to be referenced as weak in the operation block (even though it's just used to create the data task) - because:
**transporter** ->
    operation queue ->
        operation ->
            block ->
                request ->
                    **transporter**

Reply:
`urlRequest` is a struct. 

`'weak' may only be applied to class and class-bound protocol types, not 'URLRequest'`

Reply:
cool - right!